staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/merge.go

```diff
@@ -18,12 +18,13 @@ package builder
 
 import (
 	"github.com/go-openapi/spec"
+	"k8s.io/kube-openapi/pkg/aggregator"
 )
 
 // MergeSpecs aggregates all OpenAPI specs, reusing the metadata of the first, static spec as the basis.
 // Later paths and definitions override earlier ones. None of the input is mutated, but input
 // and output share data structures.
-func MergeSpecs(staticSpec *spec.Swagger, crdSpecs ...*spec.Swagger) *spec.Swagger {
+func MergeSpecs(staticSpec *spec.Swagger, crdSpecs ...*spec.Swagger) (*spec.Swagger, error) {
```

Comment:
we cannot assume that staticSpec has all kube-apiserver paths. Assume it only has apiextension, and maybe some discovery/status paths.